### ISSUE TITLE
Load on laptop CPUs

### ISSUE THREAD

**Issue by divyanshj16** [junyanz/pytorch-CycleGAN-and-pix2pix#41]
I have an HP Notebook - 15-ac026tx with Intel i5 (5th gen), 4GB RAM, 2GB AMD Radeon graphics card.[Full Specifications](https://support.hp.com/gb-en/product/hp-15-ac000-notebook-pc-series/7771404/model/8326086/document/c04731368)
I want to run your code on my laptop. Won't it harm my laptop battery and its hardware?
I read somewhere on the Internet that training high computation models can severely harm laptops?
I just want to know is it okay if I train it on my laptop?

**Comment by junyanz** [comment]
I am not sure if PyTorch supports AMD GPUs. You can train your model on the Intel CPUs, But it will be quite slow. I haven't trained any model on an everyday use laptop by myself.

**Comment by divyanshj16** [comment]
How much time does it take on your computer?

**Comment by junyanz** [comment]
It takes 3-4 hours for pix2pix training on facades datasets with a Nvidia GTX 1080 GPU.

**Comment by divyanshj16** [comment]
Haven't you saved the model? so that we can use it without training?

**Comment by junyanz** [comment]
There are a few pix2pix and cyclegan models available. See this [post](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/10) on pre-trained models for more details.

**Comment by lxj616** [comment]
Add a hint for those who searched 'time' to get here:
'It takes 3-4 hours for pix2pix training on facades datasets with a Nvidia GTX 1080 GPU.' 

which means that if you use a desktop nvidia gpu, like GTX 745 I am using, It has a 4GB vram (50% of the GTX1080), then it shall use 6-8 hours IF ONLY CONSIDERS THE BATCH SIZE LOSS

And If you are using cpu, consider a desktop i7 with enough RAM , it is about 48-64 hours in total (800% time consuming)

for a laptop 4GB RAM, you need to reduce the batch size, which is ... even slower, and if you are using i5-laptop-version, please save your time and don't even try
